.coveragerc

```diff
@@ -21,6 +21,7 @@ omit =
         hummingbot/connector/exchange/beaxy*
         hummingbot/connector/exchange/bitfinex*
         hummingbot/connector/exchange/blocktane*
+        hummingbot/connector/exchange/btc_markets*
```

Comment:
Sorry @lt7, adding exceptions to coverage is only allowed for very particular and well justified cases. A new connector is not included in the justified exceptions.

-- Reviewed in representation of the Hummingbot Foundation Technical DAO